### DISCUSSION TITLE
What settings i need to choose to get AAC LC SBR?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#432] [Q&A] [answered]
![Screenshot_10](https://user-images.githubusercontent.com/13344308/104104676-5204ac80-5288-11eb-830d-e56f4576d49e.png)
![Screenshot_11](https://user-images.githubusercontent.com/13344308/104104808-23d39c80-5289-11eb-81fd-31b7e64f1d08.png)

I wonder what settings i should choose to get these result in audio?

I'm using **BETA version 2.1.5.5**

Thx in advance!

**UPDATE:**

I got it, i don't know how to get the part of **"Spectral Band Replication"**.

**Comment by ZeroCool22** [comment, marked as answer]
Finally i got it, need to choose **Constrained VBR.** 👍

**Reply by Dendraspis** [reply]
To be precise:
The ***High Efficiency*** checkbox is the right answer, but ***HE*** is not available for TVBR.   The possibility to select both is fixed in the next beta.  😊
